wezterm-gui/src/overlay/selector.rs

```diff
@@ -222,6 +254,9 @@ impl SelectorState {
                     key: KeyCode::Backspace,
                     ..
                 }) => {
+                    if !self.filtering && !self.selection.is_empty() {
```

Comment:
pop is safe to call even when empty

```suggestion
                    if !self.filtering {
```